content/00.build/40.tooling/20.hardhat/20.guides/20.migrating-to-zksync.md

```diff
@@ -3,6 +3,16 @@ title: Migrating Hardhat project to ZKsync Era
 description: Learn how to migrate an existing Hardhat project to ZKsync Era.
 ---
 
+## Prerequisites
+
+### Node.js and package manager
```

Comment:
what about package manager? if we do not specify it, maybe it's best to remove the `h3` in this section and just list the node and hardhat versions inside Prerequisites